game_api.txt

```diff
@@ -250,6 +250,17 @@ Give Initial Stuff API
 ^ str is a comma separated list of initial stuff
 ^ Adds items to the list of items to be given
 
+Nyancat API
+-----------
+
+`nyancat.place(pos, facedir, length)`
+
+^ Place a cat at `pos` facing `facedir` with tail length `length`
```

Comment:
Just a nit, I feel like you should mention the restrictions on facedir here, as well as the restrictions on the Y coordinate.